packages/DataGrid/src/DataGrid.js

```diff
@@ -376,7 +373,7 @@ const DataGrid = React.forwardRef((props, ref) => {
     if ($isActive) $isActive.classList.toggle("grid--is-blurred");
   }, []);
 
-  const handleMouseUpGrid = React.useCallback(
+  const handleMouseDownGrid = React.useCallback(
```

Comment:
this adds faster feedback to the user instead of waiting to mouse down and mouse up (a regular click), once you mouse down this will be executed. 

This might be a breaking change on some features using mouseup to test their features, please replace mouseup with `mousedown` from now on. cc: @oscarkwan and @mikrotron

Reply:
I can't find the reference, but I feel like I read somewhere that firing an action on mouse down is not accessible 🤔

Reply:
Correct.  Although it's a 2.1 criteria...
https://www.w3.org/WAI/WCAG21/quickref/#pointer-cancellation

Reply:
what does it mean is 2.1 criteria do apply in this context?

Reply:
WCAG 2.1 is the newest standard from W3C from 2018.  Most legislation still references WCAG 2.0 (level AA) from 2008 though, and we are specifically targeting that standard for HighBond, even though at some point we know the legislation (and requirements demanded by customers) will be updated to WCAG 2.1.  So we have some leeway here, but probably _best_ to be compliant with WCAG 2.1 when we can (especially for Paprika, and other shared components, like the `<GlobalNav>`).